noobaa_sa/s3_client.py

```diff
@@ -0,0 +1,309 @@
+import os
+import logging
+import tempfile
+import boto3
+from boto3.s3.transfer import TransferConfig
+from noobaa_sa.exceptions import BucketCreationFailed
+from utility.utils import generate_unique_resource_name, generate_random_files
+from botocore.exceptions import ClientError
```

Comment:
can we order the imports

Reply:
done in https://github.com/red-hat-storage/noobaa-sa-ci/pull/11/commits/bf174e4cf192f95457c6e85267009f7cacb5063a